command/certificate/certificate.go

```diff
@@ -86,6 +86,7 @@ $ step certificate uninstall root-ca.crt
 			createCommand(),
 			formatCommand(),
 			inspectCommand(),
+			handshakeCommand(),
```

Comment:
The functionality of this command is less concerned with managing certificates than the other subcommands of `certificate`. There's some semblance with `certificate inspect` and `certificate verify`, but this is more about the TLS handshake than it is about certificates. Thus instead of putting this under `certificate`, it could be an option to define a new `tls` command and put it under that. I don't have ideas for other subcommands immediately, though.

Reply:
@maraino we've discussed this during open source triage and after quite some discussion the new `tls` subcommand came out on top. 

@tashian suggested we could add a `tls inspect` (or similar) command (in addition to `tls handshake`) in the future that would probe the server more extensively than just the initial handshake, similar to e.g. https://www.ssllabs.com/ssltest/ and https://github.com/testssl/testssl.sh. 

Furthermore, we could add `ssh handshake` for the equivalent of this PR, but for SSH handshakes. All out of scope for this PR, though.

Reply:
Fixed with 6a13fa3